spec/controllers/zines_controller_spec.rb

```diff
@@ -1,9 +1,7 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.